app/decorators/user_decorator.rb

```diff
@@ -116,9 +116,9 @@ def should_acknowledge_personal_key?(session)
   end
 
   def recent_events
-    events = user.events.order('updated_at DESC').limit(MAX_RECENT_EVENTS).map(&:decorate)
+    events = user.events.order('created_at DESC').limit(MAX_RECENT_EVENTS).map(&:decorate)
```

Comment:
sorting events on the basis of `created_at` instead of `updated_at` because the `happened_at` decorator method also uses `created_at`.